atc/db/migration/migrations/1597678793_add_job_id_to_build_image_resource_caches.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+  ALTER TABLE build_image_resource_caches
+    ADD COLUMN job_id integer REFERENCES jobs(id);
+
+  UPDATE build_image_resource_caches
+  SET job_id = builds.job_id
+  FROM builds
+  WHERE builds.id = build_id;
```

Comment:
have you profiled this migration? I imagine it must sequentially scan `build_image_resource_caches`... but not every build has image_resource caches, and that table should be regularly cleaned up by the existing query in `Finish()` on `db.Build` as well as the `CleanBuildImageResourceCaches` method on `db.resourceCacheLifecycle`... so it's fast?

Reply:
I haven't profiled the migration but in a deployment like `hush-house`, there is approx 600k rows and in our prod environment there is 9k rows. I imagined the caches should be cleaned up pretty often so it shouldn't have too many rows to update?